src/builder/command.rs

```diff
@@ -4241,8 +4241,8 @@ impl Command {
                 .action(ArgAction::Help);
             if self.long_help_exists {
                 arg = arg
-                    .help("Print help information (use `--help` for more detail)")
-                    .long_help("Print help information (use `-h` for a summary)");
+                    .help("Print help information (use --help for more detail)")
+                    .long_help("Print help information (use -h for a summary)");
```

Comment:
> Also improve readability and reduce help length a bit by removing unnecessary backticks from --help help. It should be clear from context that -h and --help are literal option names.

But quoting is consistent with our error messages.  Actually, to make this more consistent, it should use single quotes.

Please revert or switch to single quotes.  If you'd like to discuss more broadly changing our quoting, let's do that in an issue.